service/src/test/java/bio/terra/tanagra/app/controller/UnauthenticatedApiControllerTest.java

```diff
@@ -0,0 +1,42 @@
+package bio.terra.tanagra.app.controller;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import bio.terra.tanagra.generated.model.ApiSystemVersion;
+import bio.terra.tanagra.testing.BaseSpringUnitTest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.MockMvc;
+
+@Tag("regression-test")
```

Comment:
Per above, we can run this as part of the regular test suite, not the regression tests.

Reply:
changed